lib/assets/components/causes/causeTileGrid.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:app/assets/components/grid.dart';
+import 'package:app/viewmodels/causes_view_model.dart';
+import 'package:flutter/material.dart';
+import 'package:app/assets/components/causes/causeTile.dart';
+import 'package:font_awesome_flutter/font_awesome_flutter.dart';
+import 'package:stacked/stacked.dart';
+
+class causeTileGrid extends StatelessWidget {
```

Comment:
I think the standard is pascal for class names
```suggestion
class CauseTileGrid extends StatelessWidget {
```